CommonLibSF/include/RE/B/BGSLocation.h

```diff
@@ -1,12 +1,13 @@
 #pragma once
 
 #include "RE/T/TESForm.h"
+#include "RE/T/TESFullName.h"
 
 namespace RE
 {
 	class BGSLocation :
-		public TESForm  // 00
-						// ...
+		public TESForm,  // 00
+        public TESFullName // 20// ...
```

Comment:
Either remove `// ...` at the end or move to next row (if there still are more inheritence to add)

Reply:
I'm not sure if next inheritance needs to be added here so I moved //... to the next line.